cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,32 @@
+import { generateUser } from '../support/generate';
+
 describe('Sign Up page', () => {
   it('should provide an ability to register new account', () => {
+    const user = generateUser();
+
+    cy.visit('https://react-redux.realworld.io');
```

Comment:
It's a good practice to move base URL to the cypress.json configuration file. This way, you don't have to repeat the same URL in every test and it's easier to maintain.